locksmith/src/operations/metadataOperations.ts

```diff
@@ -125,34 +127,23 @@ const fetchChainData = async (
   address: string,
   keyId: string,
   network: number
-): Promise<any> => {
-  const kd = new KeyData()
-  const data = await kd.get(address, keyId, network)
+) => {
+  const keyData = new KeyData()
+  const data = await keyData.get(address, keyId, network)
+  const { attributes } = keyData.openSeaPresentation(data)
   return {
-    ...kd.openSeaPresentation(data),
     ...data,
+    attributes,
   }
 }
 
 const defaultMappings = (address: string, host: string, keyId: string) => {
   const defaultResponse = {
     name: 'Unlock Key',
-    description: 'A Key to an Unlock lock.',
+    description:
+      'A Key to an Unlock lock. Unlock is a protocol for memberships. https://unlock-protocol.com/',
     image: `${host}/lock/${address}/icon?id=${keyId}`,
   }
-
-  // Custom mappings
-  // TODO: move that to a datastore at some point...
-  Metadata.forEach((lockMetadata) => {
-    if (address.toLowerCase() == lockMetadata.address.toLowerCase()) {
-      defaultResponse.name = lockMetadata.name
-      defaultResponse.description = lockMetadata.description
-      defaultResponse.image = lockMetadata.image || defaultResponse.image
-    }
-  })
-
```

Comment:
Removed this since it was old and we store metadata for lock now so not sure local files are useful.

Reply:
Actually we may want to keep it because some old locks are still using this for their metadata and I don't think we can break them :/

Reply:
Restoring.

Reply:
thx!